spec/classes/postfix_spec.rb

```diff
@@ -224,71 +224,71 @@
           context 'when specifying a custom mastercf_content' do
             let(:params) do
               {
-                mastercf_content: 'testy',
+                'mastercf_content' => 'testy',
               }
             end
 
-            it 'does stuff' do
+            it 'expected to do stuff' do
```

Comment:
The recommended way to do it is to have declarative sentences ("it 'does stuff'"). Please don't change this.